frontend/src/components/petPlant/PetPlantEditForm/index.tsx

```diff
@@ -191,6 +201,10 @@ const PetPlantEditForm = (props: PetPlantDetails) => {
             </InputWrapper>
           </Title>
           <SubTitle>{dictName}</SubTitle>
+          <ExpandedTextBox>
+            <Text>이미지 변경하기</Text>
+            <ImageButton ref={imgRef} changeCallback={fileUploadHandler} size={16} />
```

Comment:
> 마지막은 이미지 수정 버튼 자체가 라벨이어서 중복 라벨이 되지 않네요... 아이디어가 있으면 조언 부탁드립니다...!

라벨 2개에 같은 id를 줘도 잘 동작하는 것 같아요!
다만 여기 Text를 label 태그로 만들어서 useId를 줘야할 것 같은데, 밖으로 빼는 과정이 번거로울 수 있겠네요..

```suggestion
            <Text htmlFor={imageButtonId}>이미지 변경하기</Text>
            <ImageButton htmlId={imageButtonId} ref={imgRef} changeCallback={fileUploadHandler} size={16} />
```